packages/devtools_app/lib/src/error_badge_manager.dart

```diff
@@ -65,15 +102,43 @@ class ErrorBadgeManager extends DisposableController
     notifier.value = currentCount + 1;
   }
 
+  void appendInspectorErroredWidget(InspectableWidgetError error) {
+    var currentErrors = _activeErroredWidgets.value;
+    if (!currentErrors.any((e) => e.inspectorRef == error.inspectorRef)) {
+      currentErrors = [...currentErrors, error];
```

Comment:
why aren't we just adding error to currentErrors?

Reply:
I think it's because a ValueNotifier with a list won't trigger set state if the lists content changes.

Reply:
Yep! I did try to call `notifyListeners()` directly, but here's a warning saying it's not allowed outside of the Flutter package. This code has changed slightly (it's now a map not a list), but still has to do this - if you know of a better way, let me know!